packages/perl_gcstring_linebreak.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Perl_unicode_linebreak < Package
```

Comment:
This class name does not match the filename for the package.  Please remove from this PR and create a separate PR with just this package.